src/+otp/+kortewegdevries/KortewegdeVriesParameters.m

```diff
@@ -0,0 +1,30 @@
+classdef KortewegdeVriesParameters
+    %KORTEWEGDEVRIES PARAMETERS
+    %
+    properties
+
+        Domain %MATLAB ONLY: (1,2) {mustBeFinite} = [-10, 10]
```

Comment:
Please document every parameter with a comment above it, denoting what each parameter means

Reply:
Done